x-pack/test/saved_object_api_integration/common/suites/export.ts

```diff
@@ -135,7 +138,13 @@ export const createRequest = ({ type, id }: ExportTestCase) =>
 const getTestTitle = ({ failure, title }: ExportTestCase) =>
   `${failure?.reason || 'success'} ["${title}"]`;
 
-const EMPTY_RESULT = { exportedCount: 0, missingRefCount: 0, missingReferences: [] };
+const EMPTY_RESULT = {
+  excludedObjects: [],
+  excludedObjectsCount: 0,
```

Comment:
Took me a while to understand why adding the new properties at the end of the definition was still causing a failure. TIL, for some test suites, we're asserting against the stringified so the properties needs to be in the exact same order.

Reply:
🤔 @jportner do recall why we are doing a stringified test here? Can we make this more robust by sorting the keys before stringifying, and switch to using `json-stable-stringify` instead?